src/pages/Route/Create.tsx

```diff
@@ -270,7 +270,11 @@ const Page: React.FC<Props> = (props) => {
           {renderStep()}
         </Card>
       </PageHeaderWrapper>
-      <ActionBar step={step} lastStep={redirect ? 2 : 4} onChange={onStepChange} withResultView />
+      {step !== 5 ? (
+        <ActionBar step={step} lastStep={redirect ? 2 : 4} onChange={onStepChange} withResultView />
```

Comment:
Why not update Action directly?

Reply:
> Good point but why not update the ActionBar component directly?


thanks for your proposal, update actionbar is more elegant and versatile. I've learned.